.github/workflows/tests.yml

```diff
@@ -1,13 +1,11 @@
 ---
 name: Tests
-
 on:  # yamllint disable-line rule:truthy
   push:
     branches: [main]
   pull_request:
     types: [opened, synchronize]
   workflow_dispatch:
-
```

Comment:
The line length should be changed by `yamlfix` too. Look at [the docs](https://lyz-code.github.io/yamlfix/#line-length-width). Maybe it's a bug that it's not cropping it well

Reply:
If you don't feel like debugging the issue with the cropping, you can do it manually, but I wouldn't increase the line length either in `yamlfix` or in `yamllint`

Reply:
Yea, it must be a bug. I even tried setting that specifically and it did not work

Reply:
I've confirmed the bug and [opened a new issue to track it](https://github.com/lyz-code/yamlfix/issues/221). I'd say it's better to manually fix these comments so that we can merge the PR. What do you think?